src/org/omegat/core/tagvalidation/ErrorReport.java

```diff
@@ -176,7 +176,8 @@ public static String localizedTagError(TagError error) {
             return OStrings.getString("TAG_ERROR_ORPHANED");
         case UNSPECIFIED:
             return OStrings.getString("TAG_ERROR_UNSPECIFIED");
+        default:
+            return null;
```

Comment:
```suggestion
            return "Unknown tag error is occurred!";
```
we can `return null` as same as before, but it may be better to show message that is invalid and continue, rather than NPE.

Reply:
Double check with `TagError` enum class. Here is never come condition, so we should put throwing exception.